PrinterCommunication/PrinterConnectionAndCommunication.cs

```diff
@@ -1815,6 +1820,16 @@ public void RebootBoard()
 						// let the process know we canceled not ended normally.
 						CommunicationState = CommunicationStates.Disconnected;
 					}
+
+					if (ActiveSliceSettings.Instance.PrinterSelected
```

Comment:
Rather than autoconnect, use previous connection state so that connected printers reconnect